frontends/ast/simplify.cc

```diff
@@ -899,6 +899,73 @@ bool AstNode::simplify(bool const_fold, bool at_zero, bool in_lvalue, int stage,
 				if (!is_rand_reg)
 					log_warning("reg '%s' is assigned in a continuous assignment at %s:%d.%d-%d.%d.\n", children[0]->str.c_str(), filename.c_str(), location.first_line, location.first_column, location.last_line, location.last_column);
 			}
+
+			// Copy unpacked array
```

Comment:
Could we clarify that this is for copying an _entire_ unpacked array?

Reply:
Sure. I'll fix that.